src/Native/Unix/System.Security.Cryptography.Native/pal_ssl.h

```diff
@@ -87,19 +88,19 @@ enum class HashAlgorithmType : int32_t
     SSL_GOST94 = 229410,
     SSL_GOST89 = 229411,
     SSL_AEAD = 229412,
-};
+} HashAlgorithmType;
 
-enum class DataHashSize : int32_t
+typedef enum
 {
     MD5_HashKeySize = 8 * MD5_DIGEST_LENGTH,
     SHA1_HashKeySize = 8 * SHA_DIGEST_LENGTH,
     SHA256_HashKeySize = 8 * SHA256_DIGEST_LENGTH,
     SHA384_HashKeySize = 8 * SHA384_DIGEST_LENGTH,
     GOST_HashKeySize = 256,
     Default = 0,
-};
+} DataHashSize;
 
-enum SslErrorCode : int32_t
+enum SslErrorCode
```

Comment:
I think every other enum became `typedef enum { ... } Name;`, why is this one `enum Name { ... }`?

Reply:
Where C++ enum name was used in function signature, for its C counterpart, we needed to either do `typedef enum <optionalPrototypeName> {} Name` or declare from prototype / blueprint `enum <optionalPrototypeName> {} Name`.

In this PR and previous ones, I have only updated those `enum`  and `struct` to typedef which were used in some function signature and left rest with blueprint name only.
We can sweep all these `enum` and `struct` to one style regardless of where they are used. I can do it here or in the very last PR of this series where all the __cplusplus dedcode and C++ compiler flags from CMake will be cleaned up. We are one step away from that (pending only `System.Security.Cryptography.Native.Apple` under Unix after this PR).

Reply:
Good enough for me :)

Reply:
I didn't realized that System.Native had some pending items, issued https://github.com/dotnet/corefx/pull/30431. Now `S.S.C.N.Apple` is the only one :)

Reply:
This is now addressed in https://github.com/dotnet/corefx/pull/30495. I have streamlined all `typedef struct` and `typedef enum`.